Yosemite/YosemiteTests/Stores/AppSettingsStoreTests+OrdersSettings.swift

```diff
@@ -0,0 +1,132 @@
+import XCTest
+@testable import Yosemite
+@testable import Storage
+
+final class AppSettingsStoreTests_OrdersSettings: XCTestCase {
+    /// Mock Dispatcher!
+    ///
+    private var dispatcher: Dispatcher!
+
+    /// Mock Storage: InMemory
+    ///
+    private var storageManager: MockStorageManager!
+
+    /// Mock File Storage: Load data in memory
+    ///
+    private var fileStorage: MockInMemoryStorage!
+
+    /// Test subject
+    ///
+    private var subject: AppSettingsStore!
+
+    override func setUp() {
+        super.setUp()
+        dispatcher = Dispatcher()
+        storageManager = MockStorageManager()
+        fileStorage = MockInMemoryStorage()
+        subject = AppSettingsStore(dispatcher: dispatcher!, storageManager: storageManager!, fileStorage: fileStorage!)
```

Comment:
nit: I think we can assume these variables to be non-nil from their implicitly unwrapped types?

```suggestion
        subject = AppSettingsStore(dispatcher: dispatcher, storageManager: storageManager, fileStorage: fileStorage)
```